docs/static/releasenotes.asciidoc

```diff
@@ -91,6 +91,45 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha2,Logstash 8.0.0-alpha2>>
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
+[[logstash-8-19-0]]
+=== Logstash 8.19.0 Release Notes
+
+[[notable-8-19-0]]
+==== Notable issues fixed
+
+* Improve plugins remove command to support multiple plugins https://github.com/elastic/logstash/pull/17030[#17030]
+* Significantly improves write speeds to the persistent queue (PQ) when a pipeline's workers are caught up with already-written events. https://github.com/elastic/logstash/pull/17789[#17789]
+* Eliminated log warning about unknown gauge metric type when using pipeline-to-pipeline. https://github.com/elastic/logstash/pull/17721[#17721]
+
+[[featured-8-19-0]]
+==== New features and enhancements
+
+Logstash OSS and Full docker images are now based on Ubuntu 24.04.
```

Comment:
```suggestion
Logstash OSS and Full docker images are now based on Ubuntu 24.04.
In addition to being able to ingest data from elasticsearch via the DSL, it is now possible to use ES|QL via the logstash-input-elasticsearch plugin.
```

Reply:
How about something like 
```suggestion
[[esql-8.19.0]]
===== ES|QL Support in Elasticsearch Input (Technical Preview)

Support for using ES|QL queries in the {logstash-ref}/plugins-inputs-elasticsearch.html[Elasticsearch input] to add improved flexibility when ingesting data from {es} is now in Technical Preview. 
```

(note that the `Logstash OSS and Full docker images are now based on Ubuntu 24.04.` is already in the release notes for `8.18.4`)